dojo/home/views.py

```diff
@@ -52,6 +54,16 @@ def dashboard(request: HttpRequest) -> HttpResponse:
     else:
         unassigned_surveys = None
 
+    if not settings.SHOW_LOGIN_FORM:
+        banner_config = BannerConf.objects.get()
```

Comment:
`BannerConf.DoesNotExist` and `BannerConf.MultipleObjectsReturned` should be caught, in case the BannerConf object doesn't exist or there are more than one.

Reply:
Good call. I added some unit tests around the logic as well so it's not as fragile